README.md

```diff
@@ -29,13 +29,10 @@ Just build the image
     mvn -DskipTests -DskipITs -DdockerCompose.skip
 ```
 
-## Events
+## Internal Events
 
 ### Sample Load
-Queue name: `supportTool.caseProcessor.sample`
-Exchange: N/A
-Routing key: N/A
-
+Topic: `rm-internal-telephone-capture`
```

Comment:
Think this should be `event_case-update` instead?

Reply:
fixed